tests/foreman/api/test_operatingsystem.py

```diff
@@ -140,6 +111,7 @@ def test_positive_end_to_end_os(self, module_target_sat, module_org, module_arch
         new_template = module_target_sat.api.ProvisioningTemplate(
             organization=[module_org]
         ).create()
+        new_os_parameters = [{'name': gen_string('alpha'), 'value': gen_string('alpha')}]
```

Comment:
What you think if we update this param value only instead of param itself and verify it with update?
Also, in the end before deleting OS, delete the OS param and verify its removed from OS, wdyt?

Reply:
We can go for updating only value as well, but I prefer updating both.
Added deleting OS param and is verification before OS delete